selectors.yml

```diff
@@ -155,6 +158,9 @@ selectors:
   432201:
     selector: 1458281248224512906 
     name: "avalanche-subnet-dexalot-testnet"
+  717160:
+    selector: 4418231248214522936 
+    name: "ethereum-testnet-polygon-validium-1"
```

Comment:
There's no `ethereum-testnet` called `polygon`; if `polygon-validium` is the `1`st instance of this rollup, then it must be attached to some ethereum-testnet, and that needs to be specified here

Reply:
it is a sepolia testnet chain at a high level so changed it to `ethereum-testnet....` 
https://github.com/smartcontractkit/chain-selectors/pull/41/commits/292842310ccbf8bbb9d6d5ed3baf8b2d25c63ea3